.github/workflows/full-downstream.yml

```diff
@@ -0,0 +1,65 @@
+
+name: Build Chain
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - main
+      - 8.*
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
+      - '**.md'
+      - '**.adoc'
+      - '*.txt'
+      - '.ci/**'
+
+jobs:
+  concurrency:
+    group: pull_request-${{ github.head_ref }}
+    cancel-in-progress: true
```

Comment:
I think this part should go unto the `build-chain` job; if I remember correctly, this is the configuration issue we had on the `main` branch.
Also, the group should be different than what we use in the `pull-request.yml`.

Reply:
this is set like that on `main`: https://github.com/kiegroup/optaplanner/blob/main/.github/workflows/full-downstream.yml#L23

Reply:
oh right it should be inside the build-chain. let me correct that

Reply:
cherry-picking solved it